src/Makefile

```diff
@@ -134,6 +134,7 @@ install-$(TARGET):
 install-$(TARGET)-kmip: install-$(TARGET)
 	@echo " Installing $(TARGET)-kmip"
 	install -m 755 $(TARGET)-kmip /usr/sbin/$(TARGET)-kmip
+	install -m 644 ../lib/udev/rules.d/63-sedcli.rules /usr/lib/udev/rules.d/
```

Comment:
**_/etc/udev/rules.d/_** is reserved for custom made rules.
Also, I don't see **udev** under **/user/lib/** on my linux machine.

Reply:
This leads to an error when installing kmip.
 **_Installing sedcli-kmip
install -m 755 sedcli-kmip /usr/sbin/sedcli-kmip
install -m 644 ../lib/udev/rules.d/63-sedcli.rules /usr/lib/udev/rules.d/
install: target '/usr/lib/udev/rules.d/' is not a directory: No such file or directory
Makefile:135: recipe for target 'install-sedcli-kmip' failed
make: *** [install-sedcli-kmip] Error 1_**

Reply:
Yep. This makes sense to me. I've updated this to /etc/udev/rules.d/ directory.